Userland/Libraries/LibC/time.cpp

```diff
@@ -91,8 +91,17 @@ char* ctime_r(time_t const* t, char* buf)
 
 static int const __seconds_per_day = 60 * 60 * 24;
 
-static void time_to_tm(struct tm* tm, time_t t)
+static struct tm* time_to_tm(struct tm* tm, time_t t)
 {
+    // Note: these correspond to the number of seconds from epoch to the dates "Jan 1 00:00:00 -2147483648" and "Dec 31 23:59:59 2147483647",
+    // respectively, which are the smallest and biggest representable dates without overflowing tm->tm_year, if it is an int.
+    time_t constexpr smallest_possible_time = -67768040609740800;
+    time_t constexpr biggest_possible_time = 67768036191676799;
```

Comment:
Minor thing, we always put `constexpr` on the left-hand side of the expression:
```c++
constexpr time_t smallest_possible_time = -67768040609740800;
constexpr time_t biggest_possible_time = 67768036191676799;
```

Reply:
Fixed. :^)